js/ui/scheduler/utils.js

```diff
@@ -52,14 +52,22 @@ const isTimezoneChangeInDate = (date) => {
     return (startDayDate.getTimezoneOffset() - endDayDate.getTimezoneOffset()) !== 0;
 };
 
+const isDifferentDates = (startDate, endDate) => {
+    // NOTE: subtract 1 millisecond to avoid 00.00 time
```

Comment:
I don't think we need this comment.
@andrewmakarov @Krijovnick ?

Reply:
I think comment can help.

Reply:
Ok, how about make this comment more informative?
// NOTE: Appointment endDate is "YYYY:MM:DD 00:00:00" instead of "YYYY:MM:DD 23:59:59" for a regular Date